index_api-v2.md

```diff
@@ -0,0 +1,191 @@
+# Murmurations Index API
+
+> :link: BASE URL
+>
+> _Test Environment_  
+> https://test-index.murmurations.network2/{endpoint}
+>
+> _Production Environment_  
+> https://index.murmurations.network/v2/{endpoint}
+
+The Index API enables nodes, using predefined schemas, to add, update and delete their data in the index so that aggregators can discover them.
+
+Schemas define collections of data fields that nodes can fill in to share information about themselves. Think of a schema as a form template and each instance of a completed form as a profile.
+
+Nodes store their profiles on their website or through a dedicated profile host and then request them to be referenced in the index. The index validates that the profile meets the requirements of its associated schema(s). The index does not actually store the entire profile; it is only required to store the URL of the profile's location on the web (`profile_url`) and the name(s) of the schema(s) that profile is/are based upon and must be validated against (`linked_schemas`).
+
+Using these two pieces of information, aggregators can then search the index for nodes with profiles that match specific schemas they want to use to create maps and directories, and the index returns a list of URLs (`profile_url`s) for the matched nodes' profiles. Aggregators then download from each node it's profile data for use in their apps.
+
+The index also stores additional data to enable aggregators to locate nodes based on other information in addition to their associated schemas. This additional information includes, for example:
```

Comment:
My suggestion as an edit for the above three paragraphs (suggesting this because it avoids the split between some index fields and others, which is a bit obsolete at this point, and adds the primary URL).

Nodes store their profiles on their website or through a dedicated profile host and then request them to be referenced in the index. The index validates that the profile meets the requirements of its associated schema(s). The index does not store the entire profile, only certain fields that can then be used by aggregators to find profiles that match various search criteria. 

Information stored by the index includes, for example:
 - _Linked schemas_ - the names of the schemas that the profile has been built and validated against
 - _Profile URL_ - the URL where the profile is stored
 - _Primary Node URL - the website address of the node

Reply:
I've incorporated your suggestion. Please see my next commit and let me know if its OK.